public/controllers/management/components/management/status/actions-buttons-main.js

```diff
@@ -88,7 +102,17 @@ class WzStatusActionButtons extends Component {
       this.showToast('success', 'Restarting manager.', 3000);
     } catch (error) {
       this.setState({ isRestarting: false });
-      this.showToast('danger', `Error restarting manager: ${error.message || error}`, 3000);
+      const options = {
+        context: `${WzStatusActionButtons.name}.restartManager`,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
+          error: error,
+          message: `Error restarting manager: ${error.message || error}`,
+          title: error.name || error,
```

Comment:
**question**: Isn't the `title` property used as the title of the toast? We want to display the `error.name` or `error` there instead of a more user-friendly message?